test11.js

```diff
@@ -0,0 +1,10 @@
+var foo = '' ;
+
+if(foo  === 'bar') {
```

Comment:
Codacy has a fix for the issue: [Multiple spaces found before '==='.](https://app.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=26941)
```suggestion
if(foo === 'bar') {
```